src/routes/(admin)/tests/test-[type]/TestReportDialog.svelte

```
@@ -0,0 +1,122 @@
+	<Dialog.Content class="max-w-sm">
+		<Dialog.Header>
+			<Dialog.Title>Test Report</Dialog.Title>
+		</Dialog.Header>
+		<hr class="border-border" />
+
+		<div class="flex flex-col gap-2">
+			{#if loading}
+				<div class="flex flex-col gap-2">
+					{#each [1, 2, 3] as _}
+						<div class="bg-muted h-12 animate-pulse rounded-lg"></div>
+					{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)